Formula/hueadm.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Hueadm < Formula
+  desc "Command-line management interface to phillips hue"
+  homepage "https://github.com/bahamas10/hueadm"
+  url "https://registry.npmjs.org/hueadm/-/hueadm-1.2.0.tgz"
+  sha256 "8b5936a84f112ce1d1ca9a2a917223755413e3dfa3854088fb4f2c01e2470e72"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "ECONNREFUSED", shell_output("#{bin}/hueadm -U test -H 127.0.0.1 rules 2>&1 || echo 'failed'")
```

Comment:
Does this return a non-zero exit code? If it does, you can just pass the exit code as the second argument to `shell_output`, so that you don't need to try to mislead the test with the `|| echo 'failed'` snippet.

Reply:
I'm not sure how actively maintained it is. It's just the only utility that does the job I need it to do, and I'm trying to move as many of my new-Mac-installation needs to Homebrew as possible, so I figured it was worth adding a formula in case someone else might also want it.

Anyway, updated the test per your feedback. Thanks.